jordan_bundy/README.md

```diff
@@ -0,0 +1,9 @@
+
+MY INSTRUCTIONS:
+</br>
+launch `node server.js`
+</br>
+send a post request to localhost:3000/data
+</br>
+check the file at data/input.json, it will include your input
+
```

Comment:
It wont let me comment below - your .keep should actually be named .gitkeep (it's a little more informative about why you have this empty, hidden file in the directory)